go.mod

```diff
@@ -10,7 +10,8 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/helmwave/asciigraph v1.0.0
 	github.com/helmwave/logrus-emoji-formatter v1.1.0
-	github.com/invopop/jsonschema v0.8.0
+	github.com/invopop/jsonschema v0.12.0
```

Comment:
I think I've already tried to update it and there was some backward incompatible changes. Could you please check that schema will be the same?